secboot/secboot_tpm.go

```diff
@@ -382,6 +386,49 @@ func ResealKeys(params *ResealKeysParams) error {
 	return sbUpdateKeyPCRProtectionPolicyMultiple(tpm, params.KeyFiles, authKey, pcrProfile)
 }
 
+func TpmPrepare(dir string) error {
```

Comment:
I think this function name needs to be more specific, since it is only used to "prepare" the TPM in one specific mode of operation and it uses the srk.tmpl file which AFAICT we don't use in any other mode

Reply:
But it is a generic operation, as defined in the TPM v2 spec. Storage Primary Key (SRK) Templates. It is a generic thing to import anything that has been externally pre-sealed against a given TPM. Nothing about this is specific to a given mode, it is a generic way to import externally pre-sealed secrets for this TPM. Any sort of license / decryption keys can be done this way.